src/codeql.ts

```diff
@@ -245,9 +247,22 @@ async function outputMd(
   );
   return [md];
 }
+/**
+ * Checks if the query kind is compatible with SARIF output.
+ */
+function queryCanHaveSarifOutput(compatibleQueryKinds: string[]): boolean {
+  if (
```

Comment:
I know I wrote it like this, but it would be slightly cleaner to remove the explicit `return true` and instead just do
```typescript
return compatibleQueryKinds.includes("Problem") || compatibleQueryKinds.includes("PathProblem");
```